src/schema/posts.ts

```diff
@@ -264,6 +274,21 @@ export const typeDefs = /* GraphQL */ `
     Auto generated summary
     """
     summary: String
+
+    """
+    Whether the user is the author
+    """
+    isAuthor: Int
+
+    """
+    Whether the user is the scout
+    """
+    isScout: Int
+
+    """
+    Whether post is deleted
+    """
+    deleted: Boolean
```

Comment:
Why did we add this?
I'm not sure this should be publicly available. If post is deleted we can just return not found can't we?

Reply:
But I didn't see where we plan to use it. We can return the proper to the client without the actual post element

Reply:
You can add the `deleted` field to the required columns in this case. It's a workaround but that's good enough for now